packages/assets-controllers/src/CurrencyRateController.test.ts

```diff
@@ -44,7 +44,7 @@ function flushPromises(): Promise<unknown> {
 
 describe('CurrencyRateController', () => {
   beforeEach(() => {
-    jest.useFakeTimers();
```

Comment:
Changed because Jest v27 switches to the "modern" fake timers by default and these must mock more things related to timers than the legacy fake timers, because now any of the tests that involve HTTP requests are timing out. I suspect because this is because [`node-fetch` calls `setTimeout` on the request object during the request-response flow](https://github.com/node-fetch/node-fetch/blob/7b86e946b02dfdd28f4f8fca3d73a022cbb5ca1e/src/index.js#L139), but I didn't look beyond that.

Reply:
More information on this: https://github.com/nock/nock/issues/2200

It seems that using the legacy implementation is the best solution for now. Jest v28 offers more options though. When we update to v28 we should consider using the modern timers but configuring it to leave whatever we need here unmocked.